libs/community/langchain_community/graph_vectorstores/cassandra.py

```diff
@@ -32,6 +32,7 @@ def __init__(
         session: Optional[Session] = None,
         keyspace: Optional[str] = None,
         setup_mode: SetupMode = SetupMode.SYNC,
+        **kwargs: Any,
```

Comment:
@cbornet with this change, we could drop most (all?) of the optional arguments (eg., node_table, session, keyspace, etc.) and just let the `kwargs` pass them through. That would let the underlying `GraphStore` drive more of the options. Thoughts?

Reply:
I think keeping the arguments explicit is better for documentation, code completion, ...?